Index: Bundles/ALSA/GNUmakefile.preamble
===================================================================
--- Bundles/ALSA/GNUmakefile.preamble	(revision 3880)
+++ Bundles/ALSA/GNUmakefile.preamble	(working copy)
@@ -25,7 +25,3 @@
 ADDITIONAL_INCLUDE_DIRS += $(ALSA_CFLAGS)
 BUNDLE_LIBS += $(ALSA_LIBS)
 ADDITIONAL_OBJCFLAGS += -Wall -Wno-import
-
-ifeq ($(debug), yes)
-ADDITIONAL_LDFLAGS += -lmcheck
-endif
